draft-ietf-tls-tls13.md

```diff
@@ -3240,8 +3267,8 @@ receive the server's Finished, only then sending the EndOfEarlyData
 message, followed by the rest of the handshake.
 In order to avoid deadlocks, when accepting "early_data",
 servers MUST process the client's ClientHello and then immediately
-send the ServerHello, rather than waiting for the client's
-EndOfEarlyData message.
+send their ServerHello, rather than waiting for the client's
+EndOfEarlyData message before sending their ServerHello.
```

Comment:
I tend to use "its" here.

Reply:
"servers" is plural, though.  But that's easy to fix